pkg/util/iptables/iptables.go

```diff
@@ -120,6 +120,56 @@ const MinCheckVersion = "1.4.11"
 const MinWaitVersion = "1.4.20"
 const MinWait2Version = "1.4.22"
 
+// no-op implemenatation of Interface
+type fake struct{}
+
+func NewFake() *fake {
```

Comment:
I'm not sure how I feel about this being in the iptables package - now this gets linked into real binaries, but it's 100% waste.  Shouldn't this be in your Hollow code?